website/src/pages/pricing.tsx

```diff
@@ -32,33 +32,32 @@ export default ((props: any) => (
             </Helmet>
             <div className="pricing-page">
                 <ContentSection color="primary" className="hero-section text-center py-5">
-                    <h1>Sourcegraph pricing</h1>
-                    <h3>
-                        Free to start. No commitment.
-                        <br />
-                        Only pay for active users.
-                    </h3>
+                    <h2>Universal Code Search, for less than the cost of one bad line of code.</h2>
+                    <h3>Get started in minutes for free. Scales massively.</h3>
                 </ContentSection>
                 <div className="container-fluid pricing-page__plans">
                     <div className="row pt-6">
                         <div className="col-6 col-md-3 mx-auto mb-4">
                             <PricingPlan
                                 className="pricing__plan"
-                                name="Free"
+                                name="Starter"
                                 description="For individuals and small teams."
-                                price="Free"
+                                price="$0 (0-10 users), $10 (11-25 users)"
```

Comment:
@christinaforney when this is merged can we get help with automated billing for this? I'd really hate to have to go invoice a bunch of companies for $10. We have a lot of the infrastructure set up already (we had a full online self-serve billing at one point on an old pricing model), but someone has to go update our Stripe, make sure the billing UI works, that it correctly charges for each tier on the right periods, etc.

Reply:
Yes, absolutely! Do we have a sense of how many existing companies will be impacted by this change?

Reply:
This $10 plan is not a part of this first stage of pricing, so not relevant now.